intake/container/__init__.py

```diff
@@ -0,0 +1,16 @@
+from .base import BaseContainer
+from .dataframe import DataFrame
+from .ndarray import NdArray
+from .semistructured import SemiStructured
+
+container_map = {
+    'dataframe' : DataFrame,
+    'python' : SemiStructured,
```

Comment:
This name is misleading: it is a sequence of arbitrary python objects; in the common case those objects are homogeneous, probably dicts, but that is not a requirement.

Reply:
Yeah, I agree.  I struggled with the name here.  I certainly didn't want to use `Python`.  SemiStructure came from the docs.  In either case, the naming should be consistent.